finished/wrangler.jsonc

```diff
@@ -31,26 +18,20 @@
       }
     ]
   },
-
-  // Environment variables
   "vars": {
     "APP_NAME": "applywize"
   },
-
-  // Migrations configuration
   "migrations": [
     {
       "tag": "v1",
-      "new_classes": ["SessionDurableObject"]
+      "new_sqlite_classes": ["SessionDurableObject"]
     }
   ],
-
-  // D1 database configuration
   "d1_databases": [
     {
       "binding": "DB",
-      "database_name": "applywize",
-      "database_id": "a78b63d7-768a-4479-b579-c870d14ce2bd"
+      "database_name": "__change_me__",
+      "database_id": "__change_me__"
```

Comment:
`pnpm release` will create us one automatically, but we can revert this if need be